bugsnag-plugin-android-anr/src/main/java/com/bugsnag/android/AnrPlugin.kt

```diff
@@ -4,8 +4,14 @@ import java.nio.ByteBuffer
 
 internal class AnrPlugin : BugsnagPlugin {
 
+    companion object {
+        private const val INFO_POLL_THRESHOLD_MS: Long = 2500
```

Comment:
This might need to be a shorter threshold. I've seen the dialog pop in ~1 second to as many as 12 seconds (when the device was overheating from other processes).